use-cases/ai-agent.ipynb

```diff
@@ -0,0 +1,560 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {
+    "id": "3kMALXaMv-MS"
+   },
+   "source": [
+    "# Build an AI Agent with MongoDB Atlas"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "This notebook is a companion to the [Build AI Agents with MongoDB Atlas](https://www.mongodb.com/docs/atlas/atlas-vector-search/build-agents) page. For a more traditional Python development example and detailed explanations of the code, refer to the tutorial on the page.\n",
+    "\n",
+    "This notebook demonstrates an AI agent that uses MongoDB Atlas as the database for both agentic RAG and agent memory.\n",
+    "\n",
+    "<a target=\"_blank\" href=\"https://colab.research.google.com/github/mongodb/docs-notebooks/blob/main/use-cases/ai-agent.ipynb\">\n",
+    "  <img src=\"https://colab.research.google.com/assets/colab-badge.svg\" alt=\"Open In Colab\"/>\n",
+    "</a>"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Set up the environment"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {
+    "colab": {
+     "base_uri": "https://localhost:8080/"
+    },
+    "id": "cxTXczeTghzU",
+    "outputId": "ae3a81b2-cba6-42fc-f593-8646bff77b14",
+    "vscode": {
+     "languageId": "shellscript"
+    }
+   },
+   "outputs": [],
+   "source": [
+    "pip install --quiet --upgrade pymongo voyageai openai langchain langchain_mongodb"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "import os\n",
+    "\n",
+    "os.environ[\"VOYAGE_API_KEY\"] = \"<voyage-api-key>\"\n",
```

Comment:
Nice - I really appreciate having these defined up-front, so I can just supply them and not miss them later when executing the notebook! 🎉